src/scripts/main.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('tbody');
+const buttonAddRow = document.querySelector('.append-row.button');
+const buttonAddColumn = document.querySelector('.append-column.button');
+const buttonRemoveColumn = document.querySelector('.remove-column.button');
+const buttonRemoveRow = document.querySelector('.remove-row.button');
+
+document.addEventListener('click', () => {
+  const columns = document.querySelector('tr').children.length;
```

Comment:
This line assumes that there is always at least one 'tr' element in the document. If there are no rows initially, this will throw an error. Consider adding a check to ensure that 'tr' elements exist before accessing their children.